src/main/java/org/roaringbitmap/longlong/LongIteratorFlyweight.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * (c) the authors Licensed under the Apache License, Version 2.0.
+ */
+
+package org.roaringbitmap.longlong;
+
+/**
+ * Fast iterator minimizing the stress on the garbage collector. You can create one reusable
+ * instance of this class and then {@link #wrap(RoaringTreeMap)}
+ * 
+ * For better performance, consider the {@link RoaringTreeMap#forEach} method.
+ * 
+ * @author Borislav Ivanov
+ **/
+public class LongIteratorFlyweight implements PeekableLongIterator {
+
+  @Override
+  public boolean hasNext() {
+    // TODO Auto-generated method stub
+    return false;
+  }
+
+  @Override
+  public long next() {
+    // TODO Auto-generated method stub
+    return 0;
+  }
+
+  @Override
+  public void advanceIfNeeded(int minval) {
+    // TODO Auto-generated method stub
+    
+  }
+
+  @Override
+  public long peekNext() {
+    // TODO Auto-generated method stub
+    return 0;
+  }
+
+  @Override
+  public PeekableLongIterator clone() {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
+  public void wrap(RoaringTreeMap bitmap_c) {
+    // TODO Auto-generated method stub
+    
+  }
+
+//  private int hs;
```

Comment:
I went lazy on this. I remove the class as I've just introduced to follow some ports of existing JMH benchmarks.